rb_ws/src/buggy/scripts/auton/autonsystem.py

```diff
@@ -193,8 +207,13 @@ def local_controller_thread(self):
             self.rosrate_controller.sleep()
 
     def local_controller_tick(self):
-        with self.lock:
-            self_pose, self_speed = self.get_world_pose_and_speed(self.self_odom_msg)
+        if not self.use_gps_pos:
```

Comment:
If we switch to this code, there should be a way to alert on the terminal output or on Foxglove so that this change doesn't go unnoticed by the auton system.

Reply:
Ok sounds good